webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -510,8 +510,7 @@ def test_handler404(self):
         url = reverse('home:home', args=('Unknown',))
         response = self.client.get(url)
         assert response.status_code == 404
-        assert 'Page Not Found' in response.content
-        assert 'id="products_select"' not in response.content
+        assert 'Missing product' in response.content
```

Comment:
I think this test is meant to test the generic 404 handler and just happened to be using the homepage. We should probably change the URL to something more reliably returning a 404, and add a separate test somewhere for the new 404 page.

Reply:
Oh, that's a good point. I definitely broke this. I'll fix it.